roundabout/search/views.py

```diff
@@ -300,7 +300,6 @@ def get_avail_fields():
                         dict(value="old_serial_number", text="Old Serial Number", legal_lookups='STR_LOOKUP'),
                         dict(value="location__name",      text="Location", legal_lookups='STR_LOOKUP'),
                         dict(value="build__assembly__name",  text="Build", legal_lookups='STR_LOOKUP'),
-                        dict(value="detail",       text="Notes & Details", legal_lookups='STR_LOOKUP'),
```

Comment:
This should stay. We often want to search on the "detail" field, for all of history.

Reply:
Removed again due to functionality issues & desire to take better approach... see comment below.